trophic_levels-biomass_indicators.rmd

```diff
@@ -51,18 +51,23 @@ metaData <- data.frame(Ecosystem,
 knitr::kable(metaData)
 ```
 
-
 <!-- Don't remove these three html lines -->
-<br />
-<br />
-<hr />
 
+<br /> <br />
+
+<hr />
 
 <!-- Document you work below. Try not to change  the headers too much. Data can be stored on NINA server. Since the book is rendered on the R Server this works fine, but note that directory paths are different on the server compared to you local machine. If it is not too big you may store under /data/ on this repository -->
 
 ## Introduction
 
-Here we will develop indicators of ecological condition for Norway, based on the distribution of biomass between trophic levels – specifically between vegetation and vertebrate consumers. This includes three trophic levels, plants, herbivores and carnivores. Indicators of trophic interactions are based on deviation between the observed biomass at each consumer trophic level from the expected biomass based on the biomass at the prey trophic level. The expected biomass estimates are derived from global relationships between plants and vertebrate herbivores ([Fløjgaard et al. 2022](https://doi.org/10.1111/1365-2664.14047)) and between vertebrate prey and predators ([Hatton et al 2015](https://doi.org/10.1126/science.aac6284)).
+Here we will develop indicators of ecological condition for Norway, based on the distribution of biomass between trophic levels -- specifically between vegetation and vertebrate consumers. This includes three trophic levels, plants, herbivores and carnivores. Indicators of trophic interactions are based on deviation between the observed biomass at each consumer trophic level from the expected biomass based on the biomass at the prey trophic level. The expected biomass estimates are derived from global relationships between plants and vertebrate herbivores ([Fløjgaard et al. 2022](https://doi.org/10.1111/1365-2664.14047)) and between vertebrate prey and predators ([Hatton et al 2015](https://doi.org/10.1126/science.aac6284)).
+
+Indicators developed are:
+
+**Deviation from expected vertebrate herbivore biomass** (in total, and for each ecosystem)
```

Comment:
Good english names (minus the paratheses). Norwegian names are added to the wiki. I called them _beitedyr_ and _rovdyr_ for short.